pangocffi/context.py

```diff
@@ -77,3 +77,19 @@ def _set_gravity_hint(self, hint: GravityHint):
     text, and is only relevant if the gravity of the context is
     :meth:`Gravity.EAST` or :meth:`Gravity.WEST`.
     """
+
+    def load_font(self, font_description: FontDescription) -> Font:
+        """
+        Loads a font and returns it.
+
+        :param desc:
+            the Pango font description.
+        """
+        return Font.from_pointer(
+<<<<<<< HEAD
+            pango.pango_context_load_font(self._pointer,
+                                          font_description._pointer)
+=======
+            pango.pango_context_load_font(self._pointer, font_description._pointer)
+>>>>>>> f15b3c59d7c35b35dd67899c86be84a568727b6c
```

Comment:
Looks like a git merge mistake here.

```suggestion
            pango.pango_context_load_font(self._pointer,
                                          font_description._pointer)
```